plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -110,6 +111,47 @@ protected MaterializedResult getDescribeOrdersResult()
                 .build();
     }
 
+    @Test
+    @Override // Override because the regexp is different from the base test
+    public void testPredicateReflectedInExplain()
+    {
+        assertExplain(
+                "EXPLAIN SELECT name FROM nation WHERE nationkey = 42",
+                "nationkey", "bigint", "42");
+    }
+
+    @Test
+    public void testPredicatePushdown()
+    {
+        testPredicatePushdown("true", "true", true);
```

Comment:
i was thinking of tests which had two predicates, one which could be pushed down and the other which gets retained.

This one is useful too since it was missing entirely.

Reply:
Let's improve the test coverage in follow-up.